doc/source/release/1.22.0-notes.rst

```diff
@@ -59,9 +59,9 @@ value for the ``usemask`` parameter.
 Deprecations
 ============
 
-Use delimiter rather than delimitor as kwarg in mrecords
+Use delimiter rather than delimiter as kwarg in mrecords
 --------------------------------------------------------
-The misspelled keyword argument ``delimitor`` of
+The misspelled keyword argument ``delimiter`` of
```

Comment:
This is a release note about something being spelled incorrectly, so should be left alone

Reply:
Fixed. Sorry.